nav2_smoother/test/test_savitzky_golay_smoother.cpp

```diff
@@ -206,16 +206,15 @@ TEST(SmootherTest, test_sg_smoother_noisey_path)
   EXPECT_TRUE(smoother->smooth(noisey_path_refined, max_time));
 
   length = 0;
-  double non_refined_length = 0;
   for (unsigned int i = 0; i != noisey_path.poses.size() - 1; i++) {
     length += std::hypot(
       noisey_path_refined.poses[i + 1].pose.position.x -
       noisey_path_refined.poses[i].pose.position.x,
       noisey_path_refined.poses[i + 1].pose.position.y -
       noisey_path_refined.poses[i].pose.position.y);
-    non_refined_length += std::hypot(
-      noisey_path.poses[i + 1].pose.position.x - noisey_path_baseline.poses[i].pose.position.x,
-      noisey_path.poses[i + 1].pose.position.y - noisey_path_baseline.poses[i].pose.position.y);
+    // std::hypot(
```

Comment:
remove

Reply:
Removed in https://github.com/ros-planning/navigation2/pull/3831/commits/5821f39e159d3e88fc3eaa7701231332c3413212